vertical-pod-autoscaler/deploy/vpa-rbac.yaml

```diff
@@ -4,6 +4,18 @@ kind: ClusterRole
 metadata:
   name: system:metrics-reader
```

Comment:
Is this role name one that SIG Architecture have confirmed? See https://kubernetes.io/docs/reference/access-authn-authz/rbac/#referring-to-subjects for details about reserved names.

Reply:
Where do I check? This line was already here, so I assumed it was OK.